osu.Game/Overlays/NotificationOverlayToastTray.cs

```diff
@@ -24,16 +24,16 @@ namespace osu.Game.Overlays
     /// </summary>
     public partial class NotificationOverlayToastTray : CompositeDrawable
     {
-        public override bool IsPresent => toastContentBackground.Height > 0 || toastFlow.Count > 0;
+        public override bool IsPresent => toastContentBackground.Height > 0 || Notifications.Any();
 
         public override bool ReceivePositionalInputAt(Vector2 screenSpacePos) => toastFlow.ReceivePositionalInputAt(screenSpacePos);
 
         /// <summary>
         /// All notifications currently being displayed by the toast tray.
         /// </summary>
-        public IEnumerable<Notification> Notifications => toastFlow;
+        public IEnumerable<Notification> Notifications => toastFlow.Concat(InternalChildren.OfType<Notification>());
 
-        public bool IsDisplayingToasts => toastFlow.Count > 0;
+        public bool IsDisplayingToasts => Notifications.Any();
```

Comment:
Similarly here, old code looks more correct to me. Even more so here, as the member is literally called "is *displaying* toasts". If they're not in the flow, they're not being displayed anymore, are they?